Ham/DabraLibanosHamGG1/msDesc.xml

```diff
@@ -0,0 +1,1782 @@
+              <citedRange unit="page">13-23</citedRange>
+            </bibl>
+            <bibl xml:id="schneider1989">
+              <ptr target="bm:Schneider1989Ham"/>
+            </bibl>
+          </listBibl>
+        </source>
+      </recordHist>
+    </adminInfo>
+  </additional>
+  <msPart xml:id="p1" corresp="#quire1">
+    <!--    small quire -->
+    <msIdentifier>
+      <repository ref="INS0346DL"/>
+      <idno resp="#manoscrittiEritrea">Fascicolo I</idno>
+      <altIdentifier>
+        <idno resp="#manoscrittiEritrea">Fonte B</idno>
+      </altIdentifier>
+    </msIdentifier>
+    <msContents>
+      <msItem xml:id="p1_i1">
+        <locus from="1" to="2"/>
+        <title type="incomplete" ref="LIT1488GadlaQ"/>
+       <note resp="#evangeloDoro"> End of a Gadla Qirqos</note>
+      </msItem>
+    </msContents>
+    <physDesc>
+      <objectDesc>
+        <supportDesc>
+          <support><material key="parchment"></material> of a different colour and type from the one of <ref corresp="#p2" type="mspart"/>.</support>
+          <extent>
+            <locus from="1" to="12"/>
+            <dimensions>
+              <height>180</height>
+              <width>160</width>
+            </dimensions>
+          </extent>
+        </supportDesc>
+        <layoutDesc>
+          <layout writtenLines="11 17" columns="2" xml:id="layout1">
+            <locus from="1" to="2"/>
+            <locus from="11" to="12"/>
+            <p>The layout visible on the images of <locus target="#2v #11 #12"/>.</p>
+            <ab type="pricking">Pricks are visible. <ref type="item" corresp="#a23"/> apparently ignores these.</ab>
+            <ab type="ruling">Ruling is visible</ab>
+            <ab type="ruling" subtype="pattern"> Ruling pattern:
+              1-1A-1C1C/0/1C-1C/C</ab>
+          </layout>
+        </layoutDesc>
+      </objectDesc>
+      <handDesc>
+        <handNote script="Ethiopic" xml:id="h8" corresp="#p1_i1">
+          <locus from="1" to="2"/>
+          <seg>The hand of the text on the verso of <locus target="2v"/>. No images are available of the previous folio or the recto of this.
+          </seg>
+          <seg type="rubrication">Some words rubricated.</seg>
+        </handNote>
+        <handNote script="Ethiopic" xml:id="h11" corresp="#a23">
+          <locus from="11ra1" to="11rb17"/>
+          <p>The hand of the text photographed by Bausi.</p>
+        </handNote>
+        <handNote script="Ethiopic" xml:id="h12" corresp="#a24">
+          <locus from="11va1" to="11vb6"/>
+        </handNote>
+        <handNote script="Ethiopic" xml:id="h13" corresp="#a25">
+          <locus from="12va" to="12vb"/>
+        </handNote>
+      </handDesc>
+    <additions>
+      <list>
+        <item xml:id="a26">
+          <locus from="11r" to="11r"/>
+          <!--          please Nafisa could you tell me what these are and if consistent with the rest of the text on this page?-->
```

Comment:
It is a separate note by a certain Gabra Hǝywat, it has nothing to do with what follows